test/provision/terraform/caos.auto.tfvars.dist

```diff
@@ -1,633 +1,8 @@
-    username        = "ec2-user"
-    platform        = "linux"
-    python          = "/usr/bin/python3"
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-  ############################
-  # amazon linux 2023 arm64
-  ############################
-  "PREFIX:TAG_OR_UNIQUE_NAME-arm64:al-2" = {
-    ami             = "ami-07d16074c2fdf3a19"
-    subnet          = "subnet-09b64de757828cdd4"
-    security_groups = ["sg-044ef7bc34691164a"]
-    key_name        = "caos-dev-arm"
-    instance_type   = "t4g.small"
-    username        = "ec2-user"
-    platform        = "linux"
-    python          = "/usr/bin/python3"
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-  ############################
-  # windows amd64
-  ############################
-  "PREFIX:TAG_OR_UNIQUE_NAME-amd64:windows_2012" = {
-    ami             = "ami-0c5d73971003d1ab0"
-    subnet          = "subnet-09b64de757828cdd4"
-    security_groups = ["sg-044ef7bc34691164a"]
-    key_name        = "caos-dev-arm"
-    instance_type   = "t3a.small"
-    username        = "ansible"
-    platform        = "windows"
-    python          = ""
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-  "PREFIX:TAG_OR_UNIQUE_NAME-amd64:windows_2016" = {
-    ami             = "ami-03d46abfa414238dd"
-    subnet          = "subnet-09b64de757828cdd4"
-    security_groups = ["sg-044ef7bc34691164a"]
-    key_name        = "caos-dev-arm"
-    instance_type   = "t3a.small"
-    username        = "ansible"
-    platform        = "windows"
-    python          = ""
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-  "PREFIX:TAG_OR_UNIQUE_NAME-amd64:windows_2019" = {
-    ami             = "ami-077e49a076b7e3847"
-    subnet          = "subnet-09b64de757828cdd4"
-    security_groups = ["sg-044ef7bc34691164a"]
-    key_name        = "caos-dev-arm"
-    instance_type   = "t3a.small"
-    username        = "ansible"
-    platform        = "windows"
-    python          = ""
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-  "PREFIX:TAG_OR_UNIQUE_NAME-amd64:windows_2022" = {
-    ami             = "ami-0780c2aa3485f2088"
-    subnet          = "subnet-09b64de757828cdd4"
-    security_groups = ["sg-044ef7bc34691164a"]
-    key_name        = "caos-dev-arm"
-    instance_type   = "t3a.small"
-    username        = "ansible"
-    platform        = "windows"
-    python          = ""
-    tags            = {
-      "otel_role" = "agent"
-    }
-  }
-}
+ec2_prefix = "PREFIX:TAG_OR_UNIQUE_NAME"
+
+windows_ec2 = ["windows_2012", "windows_2016", "windows_2019", "windows_2022"]
+linux_ec2 = ["ubuntu22.04", "ubuntu20.04", "ubuntu16.04", "ubuntu18.04",
+"centos7", "centos8", "centos-stream", "sles-12.3", "sles-12.4", "sles-12.5", "sles-15.1", "sles-15.2", "sles-15.3",  "sles-15.4", "sles-15.5", "redhat-7.9", "redhat-7.6", "redhat-8.4", "redhat-9.0", "debian-buster", "debian-bullseye", "debian-bookworm", "al-2", "al-2023"]
```

Comment:
Should we split them by arch? I fear that right now we cannot avoid a distro in a specific arch. We have `redhat-7.9` and `redhat-7.6` and we know that one is for arm and other for amd, but it feels weird/wrong.

Reply:
Let me know if this is what you had in mind: https://github.com/newrelic/infrastructure-agent/pull/1822/commits/98954cb3b0c3c5e974f9efac0f3f65852c1dac05